internal/pkg/cli/app_show.go

```diff
@@ -112,7 +115,7 @@ func (o *showAppOpts) Execute() error {
 }
 
 func (o *showAppOpts) description() (*describe.App, error) {
-	workloadEnvs := make(map[string][]string)
+	wkldDeployedtoEnvs := make(map[string][]string)
```

Comment:
let's move this down by `var mux sync.Mutex` to make it clearer what `mux` is guarding